templates/layout-slider.php

```diff
@@ -1,7 +1,7 @@
 <?php
 namespace Netzstrategen\Gallerya;
 
-$show_navigation = count($images) >= $nav_count_min;
+$slider_image_size = has_image_size('post-thumbnail') ? 'post-thumbnail' : 'medium';
```

Comment:
What happened here? 🤔  Variable renaming, then not using it, and never show the navigation due to the missing `$show_navigation` variable.